Library/Homebrew/test/patch_spec.rb

```diff
@@ -42,16 +42,6 @@
       it { is_expected.to be_a DATAPatch }
       it(:strip) { expect(patch.strip).to eq(:p1) }
     end
-
-    it "raises an error for unknown values" do
```

Comment:
Would be good to keep all tests around and instead adjust them if e.g. the exception class varies.

Reply:
Ok.